### PR TITLE
added IsBodyHtml attribute

### PR DESCRIPTION
**Pull request by wojszub**
Added attribute IsBodyHtml for backward compatibility. Setup new message with body (Body attribute) as plain text if IsBodyHtml attribute is set to false (default value) or as html text if IsBodyHtml attribute is set to true

### CODE REVIEW

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Api/OutlookWrapper.cs [hunk 1]
```diff
@@ -78,12 +88,22 @@ public void DiscardMail()
...
-        public void NewMessageWithAttachements(string to, string subject, string body, List<string> paths)
+        public void NewMessageWithAttachements(string to, string subject, string body, List<string> paths, bool isHtmlBody)
         {
             mailItem = Application.CreateItem(OlItemType.olMailItem);
             mailItem.To = to;
             mailItem.Subject = subject;
-            mailItem.Body = body;
+
+            if (isHtmlBody)
+            {
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Api/OutlookWrapper.cs]: DRY

Comment (wojszub) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Api/OutlookWrapper.cs]: AYW

File: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookNewMessageCommand.cs [hunk 2]
```diff
@@ -34,6 +34,9 @@ public class Arguments : CommandArguments
 
             [Argument]
             public VariableStructure Result { get; set; } = new VariableStructure("result");
+
+            [Argument(Required = false, Tooltip = "Value indicating whether the mail message body is in HTML")]
+            public BooleanStructure IsBodyHtml { get; set; } = new BooleanStructure(false);
```
Comment (przegwar) [inline G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookNewMessageCommand.cs]: Discuss name of this argument with Przemek.

Comment (wojszub) [reply G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookNewMessageCommand.cs]: Przemek approved this name

**Pull request merged**